android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -42,6 +40,7 @@ fun SurveyQuestionsScreen(
     val errorMessage by viewModel.errorMessage.collectAsState()
     val coverImageUrl by viewModel.coverImageUrl.collectAsState()
     val questionUiModels by viewModel.questionContentUiModels.collectAsState()
+    val showExitDialog by viewModel.showExitDialog.collectAsState()
```

Comment:
Yup, we can manage its state here instead in VM

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/122/commits/34df858b7cdc74f4a9252b232270a14b2ef99b47